src/components/instructor/SetDetails/SetUsers.vue

```diff
@@ -118,42 +290,67 @@ export default defineComponent({
-							field: row => row.set_dates ? (row.set_dates as QuizDates).open : 0 },
-						{ name: 'due_date', label: 'Due Date', format: formatTheDate,
-							field: row => row.set_dates ? (row.set_dates as QuizDates).due : 0 },
-						{ name: 'answer_date', label: 'Answer Date', format: formatTheDate,
-							field: row => row.set_dates ? (row.set_dates as QuizDates).answer :  0 }
-					);
-				} else if (problem_set.value.set_type === 'REVIEW') {
-					columns.splice(columns.length, 0,
-						{ name: 'open_date', label: 'Open Date', format: formatTheDate,
-							field: row => row.set_dates ? (row.set_dates as ReviewSetDates).open : 0 },
-						{ name: 'closed_date', label: 'Closed Date', format: formatTheDate,
-							field: row => row.set_dates ? (row.set_dates as ReviewSetDates).closed : 0 },
+					columns.splice(columns.length, 0, {
+						name: 'open_date',
+						label: 'Open Date',
+						format: formatTheDate,
+						field: (row: MergedUserHomeworkSet) => row.set_dates.open
+					},
+					{
+						name: 'due_date',
+						label: 'Due Date',
+						format: formatTheDate,
+						field: (row: MergedUserHomeworkSet) => row.set_dates.due
+					},
+					{
+						name: 'answer_date',
+						label: 'Answer Date',
+						format: formatTheDate,
+						field: (row: MergedUserHomeworkSet) => row.set_dates.answer
+					});
+				}
+				if ((problem_set.value as unknown as MergedUserHomeworkSet).set_params.enable_reduced_scoring) {
+					columns.splice(columns.length - 1, 0,
+						{
+							name: 'reduced_scoring_date',
+							label: 'Reduced Scoring Date',
+							format: formatTheDate,
+							field: (row: MergedUserHomeworkSet) => row.set_dates ?
+								row.set_dates.reduced_scoring ?? 0 :
+								0
+						}
 					);
 				}
-				date_edit.value = { ...problem_set.value.set_dates };
+				// } else if (problem_set.value.set_type === 'QUIZ') {
+				// columns.splice(columns.length, 0,
+				// {
+				// name: 'open_date',
+				// label: 'Open Date',
+				// format: formatTheDate,
+				// field: (row: MergedUserQuiz) => row.set_dates ? (row.set_dates).open : 0
+				// },
+				// {
+				// name: 'due_date',
+				// label: 'Due Date',
+				// format: formatTheDate,
+				// field: (row: MergedUserQuiz) => row.set_dates ? (row.set_dates).due : 0
+				// },
+				// {
+				// name: 'answer_date',
+				// label: 'Answer Date',
+				// format: formatTheDate,
+				// field: (row: MergedUserQuiz) => row.set_dates ? (row.set_dates).answer :  0
+				// }
+				// );
+				// } else if (problem_set.value.set_type === 'REVIEW') {
+				// columns.splice(columns.length, 0,
+				// { name: 'open_date', label: 'Open Date', format: formatTheDate,
+				// field: (row: MergedUserReviewSet) => row.set_dates ? (row.set_dates).open : 0 },
+				// { name: 'closed_date', label: 'Closed Date', format: formatTheDate,
+				// field: (row: MergedUserReviewSet) => row.set_dates ? (row.set_dates).closed : 0 },
+				// );
+				// }
```

Comment:
Is all this to be removed? Or yet to be updated?

Reply:
I think I had pulled this out and never put it back in to test.

Reply:
fixed on next push.